.github/workflows/build.yml

```diff
@@ -679,7 +679,9 @@ jobs:
   job_e2e_tests:
     name: E2E Tests
     # We only run E2E tests for non-fork PRs because the E2E tests require secrets to work and they can't be accessed from forks
-    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository
+    if:
```

Comment:
super-l: should we update the comment that secrets also can't be accessed for dependabot PRs?

Reply:
sure, good point, why not!